diff --git a/src/Transformers/TransformerResolver.php b/src/Transformers/TransformerResolver.php
index 9b4ab4329267c1a65af20ab6017356b89788c789..492a3492d05b7f13a71a4a9a9a957fea88d7bb20 100644
--- a/src/Transformers/TransformerResolver.php
+++ b/src/Transformers/TransformerResolver.php
@@ -28,7 +28,8 @@ class TransformerResolver implements TransformerResolverInterface
      * Inspects the underlying Eloquent model to determine an appropriately
      * named transformer class, and instantiate the object.
      *
-     * @inheritdoc
+     * {@inheritdoc}
+     *
      * @throws \Rexlabs\Laravel\Smokescreen\Exceptions\UnresolvedTransformerException
      */
     public function resolve(ResourceInterface $resource)
@@ -54,11 +55,11 @@ class TransformerResolver implements TransformerResolverInterface
                     (new \ReflectionClass($model))->getShortName());
                 $transformer = resolve($transformerClass);
             } catch (\Exception $e) {
-                throw new UnresolvedTransformerException('Unable to resolve transformer for model: ' . \get_class($model),
+                throw new UnresolvedTransformerException('Unable to resolve transformer for model: '.\get_class($model),
                     0, $e);
             }
         }
 
         return $transformer;
     }
-}
\ No newline at end of file
+}
diff --git a/tests/Unit/SmokescreenTest.php b/tests/Unit/SmokescreenTest.php
index 7fc24daa094c92b68dcb8e4a71f8e2a5c2ee0646..d25dd66d96a55fb3a0d45bcf4b605597d8cf55ad 100644
--- a/tests/Unit/SmokescreenTest.php
+++ b/tests/Unit/SmokescreenTest.php
@@ -452,7 +452,8 @@ class SmokescreenTest extends TestCase
 
     public function test_default_serializer_can_be_configured_with_object()
     {
-        $obj = new class extends DefaultSerializer {};
+        $obj = new class() extends DefaultSerializer {
+        };
         $stub = $this->getMockBuilder(Smokescreen::class)
             ->setMethods(['serializeWith'])
             ->disableOriginalConstructor()
@@ -550,7 +551,6 @@ class SmokescreenTest extends TestCase
         $this->assertEquals(200, $response->getStatusCode());
     }
 
-
     public function test_can_override_the_transformer_resolver()
     {
         $this->createSchemas();
@@ -571,7 +571,6 @@ class SmokescreenTest extends TestCase
             ->resolveTransformerVia($stub)
             ->transform(User::first())
             ->toArray();
-
     }
 
     protected function createQueryBuilder(): \Illuminate\Database\Query\Builder
@@ -701,5 +700,4 @@ class SmokescreenTest extends TestCase
                 ]
             );
     }
-
 }
